src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1474,3 +1474,13 @@
 type: command
 short-summary: Run a specific triggered webjob hosted on a web app.
 """
+
+helps['webapp remote-connection'] = """
+    type: group
+    short-summary: Create a remote connection using a tcp tunnel to your web app
+"""
+
+helps['webapp remote-connection create'] = """
```

Comment:
will you ever have other commands under  `remote-connection` like `remove`? If not, then you should just create a command called `create-remote-connection` and get rid of the group.

Reply:
There is a lot of documentation for `remote-connection create` I think we need a plan or announcement before renaming it.